speakers/src/main/java/in/droidcon/speakers/presentation/ui/SpeakersAdapter.kt

```diff
@@ -29,29 +29,28 @@ class SpeakersAdapter(
 
     inner class SpeakersHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
 
-        private val speakerImgView: ImageView = itemView.findViewById(R.id.speakerImgView)
-        private val speakerNameView: TextView = itemView.findViewById(R.id.speakerNameView)
-        private val speakerOrgView: TextView = itemView.findViewById(R.id.speakerOrgView)
-
         fun bind(speakerItem: SpeakerEntity, position: Int) {
-            speakerNameView.text = speakerItem.speakerName
-            speakerOrgView.text = speakerItem.speakerOrg
-            itemView.setOnClickListener { listItemClickListener.onSpeakerItemClicked(speakerItem) }
-            Glide.with(speakerImgView.context)
-                .load(speakerItem.speakerImg)
-                .placeholder(R.drawable.ic_placeholder)
-                .circleCrop()
-                .error(R.drawable.ic_placeholder)
-                .into(speakerImgView)
+            with(itemView) {
+                speakerNameView.text = speakerItem.speakerName
+                speakerOrgView.text = speakerItem.speakerOrg
+                itemView.setOnClickListener { listItemClickListener.onSpeakerItemClicked(speakerItem) }
+                Glide.with(speakerImgView.context)
+                    .load(speakerItem.speakerImg)
+                    .placeholder(R.drawable.ic_placeholder)
```

Comment:
Since we are using `shimmer view`, we can replace this placeholder with a shimmer drawable. But this is more of a UI task. You can ignore it for now.